staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4012,7 +4012,7 @@ type ServiceSpec struct {
 	// of the service will fail. This field can not be changed through updates.
```

Comment:
How can you wipe it if it cannot be changed through updates?

Reply:
it *can* be changed iff the old service or new service in the update request was of type ExternalName:

https://github.com/kubernetes/kubernetes/blob/2318a13228dfaadd2de4669c8ce3ffb126ed3958/pkg/apis/core/validation/validation.go#L4445-L4455